spec/registrant_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+RSpec.describe Registrant do
+    before(:each) do
+        @registrant_1 = Registrant.new('Bruce', 18, true )
+        @registrant_2 = Registrant.new('Penny', 16 )
+        @registrant_3 = Registrant.new('Tucker', 15 )
+        @facility_1 = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014' })
+        @facility_2 = Facility.new({name: 'Ashland DMV Office', address: '600 Tolman Creek Rd Ashland OR 97520', phone: '541-776-6092' })
```

Comment:
We shouldn't need any facility objects to test our Registrant class. If we do, maybe our Registrant is too dependant on another class and we should decouple those.